app/assets/stylesheets/layout/_home.scss

```diff
@@ -103,6 +103,18 @@
 }
 
 
+.coming-soon {
+    width: 100%;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background, font-size, font-weight, height, padding-top, position, text-align, width, z-index